### DISCUSSION TITLE
Installation fails

### DISCUSSION THREAD

**Discussion by bmrussell** [dotnet/interactive#2171] [Q&A]
Hi - I'm failing to install .NET interative in vscode with the below error. Any ideas? I've tried installing the .visx manually and from the extensions gallery. 

Installation of the .visx appears to work, but on creating a new blank notebook I get the the toast message 
"_Installing .NET interative version 1.0.3362040"_ 
and then
"_command 'dotnet-interactive.newNotebook' not found_"

I could install globally:
```
dotnet tool install --global Microsoft.dotnet-interactive --version 1.0.336204
```
and locally:
```
dotnet new tool-manifest # if you are setting up this repo
dotnet tool install --local Microsoft.dotnet-interactive --version 1.0.336204
```
with no issues, but creating a new notebook still gives the error.

My nuget config (C:\Users\%USERNAME%\AppData\Roaming\NuGet\NuGet.Config) has the below. I changed `signatureValidationMode` from `require` to `allow` and `allowUntrustedRoot` from `false` to `true` with no effect.

```
<?xml version="1.0" encoding="utf-8"?>
<configuration>
	<config>
		<add key="signatureValidationMode" value="accept"/>
	</config>
	<packageSources>
		<add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3"/>
	</packageSources>
	<trustedSigners>
		<author name="microsoft">
			<certificate fingerprint="3F9001EA83C560D712C24CF213C3D312CB3BFF51EE89435D3430BD06B5D0EECE" hashAlgorithm="SHA256" allowUntrustedRoot="true"/>
			<certificate fingerprint="AA12DA22A49BCE7D5C1AE64CC1F3D892F150DA76140F210ABD2CBFFCA2C18A27" hashAlgorithm="SHA256" allowUntrustedRoot="true"/>
		</author>
	</trustedSigners>
</configuration>
```

If its any help the network I am on does TLS inspection, so everything has their root cert.

**Comment by jonsequitur** [comment]
Are you still seeing this issue?

Not that you don't need to install `dotnet-interactive` from the command line if you install the VS Code extension. the VS Code extension will install its own copy of the `dotnet-interactive` tool.

**Reply by bmrussell** [reply]
Hi @jonsequitur 

![image](https://user-images.githubusercontent.com/7798077/186720873-70df4804-8a72-4a4d-9868-e5df70cd761e.png)

Yes, still present. The toast remains after dismissing the dialog. It must be something specific to my work machine as it's OK at home. I've eliminated TLS inspection as the root cause. 

I could supply logs if that would help (&  knew what to send)?

**Reply by jonsequitur** [reply]
You can find the VS Code extension's logs here: 

<img width="941" alt="image" src="https://user-images.githubusercontent.com/547415/187955751-bd3df1b0-78e7-47e0-a74e-16de51542f00.png">
